packages/venia-concept/src/components/ProductOptions/getOptionType.js

```diff
@@ -0,0 +1,5 @@
+const customAttributes = {
+    fashion_color: 'swatch'
+};
+
+export default ({ attribute_code: code }) => customAttributes[code];
```

Comment:
I know it's small but a unit test here would be good.

Reply:
Coverage report has this covered. Is it still best practice to add an individual test for this file?

```
 venia-concept/src/components/ProductOptions                            |      100 |      100 |      100 |      100 |                   |
  getOptionType.js                                                      |      100 |      100 |      100 |      100 |                   |
  index.js                                                              |        0 |        0 |        0 |        0 |                   |
  option.js                                                             |      100 |      100 |      100 |      100 |                   |
  options.js                                                            |      100 |      100 |      100 |      100 |                   |
  swatch.js                                                             |      100 |      100 |      100 |      100 |                   |
  swatchList.js                                                         |      100 |      100 |      100 |      100 |                   |
  tile.js                                                               |      100 |      100 |      100 |      100 |                   |
  tileList.js                                                           |      100 |      100 |      100 |      100 |                   |
```

Reply:
I'm assuming it's covered because some other test exercises this code path.

I think it is best practice to have unit tests for any public function exported by a module. These are the discrete "units" that we want to ensure work as we expect.

If this code were to change in a breaking way, we'd know immediately via a unit test in a `getOptionType.spec.js` file, not a failing unit test in some other file that we'd then have to go debug and track it down to this function.

Further, who knows what that other file's unit test is actually checking for, so if this code were to break we may not even be alerted via a failing test.

That other code could get refactored to not use this function or something else too, but we'd still want to have confidence that it works.

Reply:
We don't need to unit test internals. I'm as guilty of this as anyone, but I say it's okay not to have an individual test per file.

Reply:
Slipped my mind that this was internal.  This definitely isn't blocking a 👍 , I was just responding to the best practice question. I'm cool with it as is 👍 .

Reply:
Yes to both. It's effectively internal and covered by other tests, so it's not necessary to have a unit test for it. But I'd like to have one anyway.